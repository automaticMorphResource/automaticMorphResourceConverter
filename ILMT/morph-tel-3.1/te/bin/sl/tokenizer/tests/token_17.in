పర్యాటకుల వల్ల ప్రభుత్వానికి బోలెడంత ఆదాయం సమకూరినా.. కొంతవరకు అక్కడ జీవించే జంతుజాలానికి మాత్రం హాని జరుగుతోంది. 
