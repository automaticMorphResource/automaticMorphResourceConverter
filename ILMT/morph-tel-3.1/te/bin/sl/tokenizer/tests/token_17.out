<Sentence id="1">
1	పర్యాటకుల	unk
2	వల్ల	unk
3	ప్రభుత్వానికి	unk
4	బోలెడంత	unk
5	ఆదాయం	unk
6	సమకూరినా..	unk
7	కొంతవరకు	unk
8	అక్కడ	unk
9	జీవించే	unk
10	జంతుజాలానికి	unk
11	మాత్రం	unk
12	హాని	unk
13	జరుగుతోంది	unk
14	.	unk
</Sentence>

